app/modules/product/card/Card.tsx

```diff
@@ -1,73 +1,20 @@
 import { FC } from 'react'
-import cl from './Card.module.scss'
-import Image from 'next/image'
-import names from 'classnames'
-import { lato } from '@/app/styles/fonts'
-import { Product } from '@/app/modules/product/types/product.type'
-import Link from 'next/link'
+import FeaturedCard from './variants/featured/FeaturedCard'
+import { Props } from './Card.interface'
+import { assertUnreachable } from '@/app/helpers'
+import BiggerCard from './variants/bigger/BiggerCard'
 
-type Props = {
-  product: Product
-}
+const Card: FC<Props> = ({ product, variant }) => {
+  switch (variant) {
```

Comment:
well we rarely use switch, instead we use `if (variant === 'featured')`, reason for this is that you may add additional parameteres inside if as well as you may have reallly big componnet in future and when you go to the next case you may not always what excatly this case for, in vase of if you always see that it belongs to the variant.